zypper-formula/zypper/packages.sls

```diff
@@ -1,13 +1,46 @@
-{% set packages = salt['pillar.get']('zypper:packages', {}) %}
+{%- set packages = salt['pillar.get']('zypper:packages', {}) %}
+{%- set defaults = namespace(refresh=False) %}
+{%- set fromdefaults = [] %}
+{%- set fromrepos = {} %}
 
-{% for package, data in packages.items() %}
+{%- for package, config in packages.items() %}
+
+{%- if 'fromrepo' in config -%}
+{%- set fromrepo = True -%}
```

Comment:
`fromrepo` seems to be unused (unless i overlooked something, only `config.fromrepo` is used)

Reply:
Right, was a leftover.